docs/source/tutorials/profiling.ipynb

```diff
@@ -5,11 +5,375 @@
+    "\n",
+    "</div>\n",
+    "\n",
+    "\n",
+    "## RADICAL Analytics\n",
+    "\n",
+    "Parsing and filtering the `*.prof`files requires a solid understanding of: \n",
+    "\n",
+    "1. RP's event/state model in order to select only the information you actually need;\n",
+    "2. the format in which the information is encoded within in `*.prof` file;\n",
+    "3. in which file or files RP recorded the information you need;\n",
+    "4. bash commands/scripts to parse and filter those files.\n",
+    "\n",
+    "That is both **difficult** and **cumbersome**. \n",
+    "\n",
+    "In order to facilitate using RP traces to profile the execution of your application, we created [RADICAL Analytics (RA)](https://radicalanalytics.readthedocs.io/en/latest/). Please see the linked documentation and consider using RA whenever you need postmortem profiling. In the following, we discuss how you can perform some task-related tracing at runtime or postmortem, i.e., while RP is executing your application or once it finished. This is useful while developing your application and/or for sport-checks while executing a large run. For production profiling, you should use RA. \n",
+    "\n",
+    "## Tracing at runtime\n",
+    "\n",
+    "In a real-life scenario, you will:\n",
+    "\n",
+    "1. Start the execution of your application on a [supported](../supported.rst) HPC platform;\n",
+    "2. while the application is running, execute shell commands to parse and filter across multiple `*.profile` files.\n",
+    "\n",
+    "The format of this tutorial does not allow for concurrently running both your application and the shell commands you need to spot-check the behavior of your tasks. Thus, we first run a sample application and then show some useful shell commands.\n",
+    "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "    \n",
+    "__Warning:__ In real-life scenarios, your shell commands will operate on multiple small files created on a filesystem shared by all the compute nodes of the HPC platform. Even with just as little as a few hundreds tasks, your shell commands could deteriorate the performance of the shared file systems. You should exercise **extreme caution** as the whole HPC machine could grind to a halt. That would affect all the users of the machine and make the system administrators very unhappy; ask as we know... \n",
+    "\n",
+    "</div>\n",
+    "\n",
+    "Here we: \n",
+    "\n",
+    "1. Use the `RADICAL_PROFILE` environment variable to enable RP tracing;\n",
+    "2. run a condensed version of the application we used for the [Getting Started](../getting_started.ipynb).\n",
+    "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "    \n",
+    "__Warning:__ If you do not set `RADICAL_PROFILE`, RP will **not** create the `*.prof` files.\n",
```

Comment:
by default `RADICAL_PROFILE` is set to `TRUE`

https://github.com/radical-cybertools/radical.utils/blob/devel/src/radical/utils/configs/utils_default.json#L9

Reply:
Confirm about default value - I didn't set it and prof-files were created, and agent side had a corresponding env variable set
```
> cat ~/radical.pilot.sandbox/re.session.polaris-login-04.matitov.019515.0009/pilot.0000/env/bs0_pre_0.env | grep RADICAL
RADICAL_BASE=/home/matitov/radical.pilot.sandbox
RADICAL_PROFILE=TRUE
RADICAL_SMT=1
```